src/interactions/map_city/enterStoreMainArea.interaction.js

```diff
@@ -0,0 +1,31 @@
+import { displayDialogue } from '../../utils';
+import { sampleStoreInteraction, sampleArcadeInteraction } from './enterMapArcade.interactions';
+
+//player interaction logic with arcade
+function sampleArcadeInteraction() {
+    // sample of coins balance
+    var coinBal = 1000;
+
+    return 'You have ${coinBal} coins to spend';
+
+};
+
+//shows what happens when player is in arcade
+export const storeMainAreaInteraction = (player, k, map) => {
+    //if user runs into the arcade, show them the balance
+    player.onCollide('enter_map_arcade', () => {
+        player.isInDialog = true;
+        displayDialogue(sampleStoreInteraction(), () => {
+            player.isInDialog = false;
+        });
+    //if user runs into the house, show them the house
```

Comment:
pleaseo remove this code because there is already a enter_map_arcade interaction. otherwise it is not possible anymore to go into the arcade